packages/dart/lib/src/objects/response/parse_exception_response.dart

```diff
@@ -22,6 +22,12 @@ ParseResponse buildParseResponseWithException(Exception exception) {
     ));
   }
 
+  if (exception is ClientException) {
```

Comment:
We use two clients, dio and http, at Parse Flutter !

In this change you are using the functions of the `http` package. Will this not cause any problems when we use the dio package client?

Reply:
`DioException` is already being handled above but `ClientException` was missing.